webrtc.html

```diff
@@ -1525,6 +1525,10 @@
                         the corresponding media description.</p>
                       </li>
                       <li>
+                        <p>If <var>transceiver</var>'s <a>[[\Stopped]]</a> slot
+                        is <code>true</code>, abort these sub steps.</p>
```

Comment:
Yes, it might be a good idea to split it up. We have a lot of conditions regarding local/remote and offer/answer. It would be nice to have separate algorithms for setLocal/setRemote and perhaps have a set of shared substeps that could be referenced.